src/hooks/useCachedRepositories.ts

```
@@ -8,6 +8,34 @@ import {
   isCacheStale
 } from '../utils/cacheUtils';
 
+const activeRefreshes = new Set<string>();
+
+/**
+ * Handles background refresh of cached data
+ */
+const handleBackgroundRefresh = async <T>(
+  cacheKey: string,
+  fetchFn: (...args: any[]) => Promise<T>,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)